lib/main.js

```diff
@@ -63,10 +63,10 @@ exports.activate = function () {
   }
 }
 
-const STYLED_REGEX = /\bstyled\b/i
+const CSS_REGEX = /\bstyled\b|\bcss\b/i
 
-function languageStringForTemplateTag (tag) {
-  if (STYLED_REGEX.test(tag)) {
+function languageStringForTemplateTag(tag) {
+  if (CSS_REGEX.test(tag.toLowerCase())) {
```

Comment:
Shouldn't need the toLowerCase since the regex is already case-insensitive. Otherwise, this looks good!